doc/content/source/bcs/DisplaceBoundaryBC.md

```diff
@@ -1,18 +1,12 @@
 # DisplaceBoundaryBC
 
-!alert construction title=Undocumented Class
-The DisplaceBoundaryBC has not been documented. The content listed below should be used as a starting point for
-documenting the class, which includes the typical automatic documentation associated with a
-MooseObject; however, what is contained is ultimately determined by what is necessary to make the
-documentation clear for users.
-
 !syntax description /BCs/DisplaceBoundaryBC
 
-## Overview
+!! ## Overview
 
 !! Replace these lines with information regarding the DisplaceBoundaryBC object.
 
-## Example Input File Syntax
+!! ## Example Input File Syntax
```

Comment:
How much work would it be to find an example input file from the regression tests? As a user I appreciate having the example linked directly in the documentation page, even if I go open that input file in VSCode immediately

Reply:
Examples added everywhere except `AbsoluteValueIndicator`, which isn't used in any input files, in e0ad9d8. Will create an issue.